state_transfer/endpoint/load_balancer/load_balancer.go

```diff
@@ -0,0 +1,105 @@
+package load_balancer
+
+import (
+	"context"
+	"time"
+
+	"github.com/konveyor/crane-lib/state_transfer/endpoint"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/apimachinery/pkg/util/intstr"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+type LoadBalancerEndpoint struct {
+	name      string
+	namespace string
+	hostname  string
+
+	labels map[string]string
+	port   int32
+}
+
+func (l *LoadBalancerEndpoint) Create(c client.Client) error {
+	err := createLoadBalancerService(c, l)
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func (l *LoadBalancerEndpoint) SetHostname(hostname string) {
+	l.hostname = hostname
+}
+
+func (l *LoadBalancerEndpoint) Hostname() string {
+	return l.hostname
+}
+
+func (l *LoadBalancerEndpoint) SetPort(port int32) {
+	l.port = port
+}
+
+func (l *LoadBalancerEndpoint) Port() int32 {
+	return l.port
+}
+
+func (l *LoadBalancerEndpoint) Name() string {
+	return l.name
+}
+
+func (l *LoadBalancerEndpoint) Namespace() string {
+	return l.namespace
+}
+
+func (l *LoadBalancerEndpoint) Labels() map[string]string {
+	return l.labels
+}
+
+func (l *LoadBalancerEndpoint) Type() endpoint.EndpointType {
+	// endpoint type is not valid for this
+	return ""
```

Comment:
since `Type()` doesn't apply to all endpoints, I believe we don't need it at the top level?

Reply:
good point, but can we save that for a separate PR? my focus this PR was to keep everything same except the refactoring of packages?